server/backend/quiz/tests/test_views.py

```
@@ -0,0 +1,81 @@
+import pytest
+from rest_framework.test import APIClient
+from rest_framework import status
+from users.models import User
+from quiz.models import Category, Question, Option, QuizAttempt, UserAnswer
```

Ruff (0.8.2)

`quiz.models.QuizAttempt` imported but unused

Remove unused import

(F401)